docworks-dts/test/__snapshots__/dts-repo.test.js.snap

```diff
@@ -1432,6 +1443,83 @@ declare namespace wix_dataset {
 
 }
 
+declare namespace wix_dot_backend_v2 {
```

Comment:
shouldn't it be a module name with `-` insed of `_` ?

Reply:
same as here
https://github.com/wix/docworks/blob/5066f336ad66d63a2bd5cc58e5c1748277e9037f/docworks-dts/test/__snapshots__/dts-repo.test.js.snap#L1523